packages/vue-components/src/questions/Question.vue

```diff
@@ -264,6 +264,11 @@ export default {
     checkMcqAnswer(markAsAnsweredIfWrong) {
       const selectedAnswer = this.answers.find(answer => answer.selected);
       if (!selectedAnswer) {
+        // shake the card
+        this.shakeClass = 'shake';
+        setTimeout(() => {
+          this.shakeClass = null;
+        }, 800);
```

Comment:
I noticed that ```markAsWrong``` also shakes the card. Wonder if you think it is worth it to refactor the function to shake the card? 
ie make a function which shakes the card which this part of the code and ```markAsWrong``` can call

Reply:
I think that's a great idea! I will update it in the next commit!